client/rpc/src/eth.rs

```diff
@@ -2249,8 +2244,14 @@ where
 	C: Send + Sync + 'static,
 	B: BlockT<Hash = H256> + Send + Sync + 'static,
 {
-	fn is_listening(&self) -> Result<bool> {
-		Ok(true)
+	fn version(&self) -> Result<String> {
```

Comment:
Why change order for this?

Reply:
It's formatted by `clion` beacuse of trait function order mismatch.